firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -217,4 +225,22 @@ public void testGetId_PersistedFidError_BackendOk() throws InterruptedException
           .isEqualTo(FirebaseInstallationsException.Status.CLIENT_ERROR);
     }
   }
+
+  @Test
+  public void testGetAuthToken_registeredFid_successful() throws Exception {
```

Comment:
I just added one test to review the IntDef usage. I will add more tests, once that is reviewed . Thank you.